### PR TITLE
deprecate error-view and no-data view from repository

### PR DESCRIPTION
**Pull request by kschuste**
### Summary
This resolves issue #252.
deprecating terra-error-view and terra-no-data-view.

### Additional Details
terra-error-view and terra-no-data-view have been replaced by the terra-status-view in the terra-core project.  Therefore, can deprecate these components.

### CODE REVIEW

**Comment by mjhenkes** [comment]
I know I told you to do this, but I think i changed my mind. Can we set the packages to hidden and update the readme's for terra clinical site to say that the components are depreciated and point to terra-status? 
Heres an example of how we set terra-button 2.0 to hidden. https://github.com/cerner/terra-core/blob/master/lerna.json

**Comment by mjhenkes** [comment]
Is there an easy way to also disable running tests on these packages?

**Comment by kschuste** [comment]
@mjhenkes good idea, we are able to exclude the paths in the configuration setups to keep them from running

File: packages/terra-clinical-error-view/README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 [![NPM version](http://img.shields.io/npm/v/terra-clinical-error-view.svg)](https://www.npmjs.org/package/terra-clinical-error-view)
 [![Build Status](https://travis-ci.org/cerner/terra-clinical.svg?branch=master)](https://travis-ci.org/cerner/terra-clinical)
 
-The Terra ErrorView is a display presented with an error message and location for a button.
+The Terra ErrorView is a display presented with an error message and location for a button.  This component has been deprecated in
```
Comment (mjhenkes) [inline packages/terra-clinical-error-view/README.md]: Could we bold the 'deprecated' sentence for all these readmes?

Comment (kschuste) [reply packages/terra-clinical-error-view/README.md]: deprecated is now in bold

**Pull request merged**